lib/new_relic/control/instrumentation.rb

```diff
@@ -62,7 +62,6 @@ def _install_instrumentation
           File.join(instrumentation_path, app.to_s, '*.rb')
         @instrumentation_files.each { |pattern| load_instrumentation_files(pattern) }
         DependencyDetection.detect!
-        ruby_deprecation
         rails_32_deprecation
```

Comment:
We still have a deprecation message going out when the user is running Rails 3.2. I think this makes sense, as we haven't ripped out the code to support Rails 3.2 yet.